src/App.tsx

```diff
@@ -33,52 +36,95 @@ import '@ionic/react/css/display.css';
 
 /* Theme variables */
 import './theme/variables.css';
+import './theme/app.scss';
 import HealthApp from './pages/HealthApp';
 
 setupIonicReact();
 
-const App: React.FC = () => (
-  <IonApp>
-    <IonReactRouter>
-      <IonTabs>
-        <IonRouterOutlet>
-          <Route exact path="/tab1">
-            <Tab1 />
-          </Route>
-          <Route exact path="/tab2">
-            <Tab2 />
-          </Route>
-          <Route path="/tab3">
-            <Tab3 />
-          </Route>
-          <Route exact path="/healthapp" >
-            <HealthApp />
-          </Route>
-          <Route exact path="/">
-            <Redirect to="/tab1" />
-          </Route>
-        </IonRouterOutlet>
-        <IonTabBar slot="bottom">
-          <IonTabButton tab="tab1" href="/tab1">
-            <IonIcon icon={triangle} />
-            <IonLabel>Tab 1</IonLabel>
-          </IonTabButton>
-          <IonTabButton tab="tab2" href="/tab2">
-            <IonIcon icon={ellipse} />
-            <IonLabel>Tab 2</IonLabel>
-          </IonTabButton>
-          <IonTabButton tab="tab3" href="/tab3">
-            <IonIcon icon={square} />
-            <IonLabel>Tab 3</IonLabel>
-          </IonTabButton>
-          <IonTabButton tab="healthapp" href="/healthapp">
-            <IonIcon icon={square} />
-            <IonLabel>Health App Integration</IonLabel>
-          </IonTabButton>
-        </IonTabBar>
-      </IonTabs>
-    </IonReactRouter>
-  </IonApp>
-);
+const App: React.FC = () => {
+  if(!isPlatform("android") && !isPlatform("ios")) {
+    return (
+      <IonApp>
+        <IonReactRouter>
+          <IonRouterOutlet>
+            <Route exact path="/tab1">
+              <Tab1 />
+            </Route>
+            <Route exact path="/tab2">
+              <Tab2 />
+            </Route>
+            <Route path="/tab3">
+              <Tab3 />
+            </Route>
+            <Route exact path="/healthapp" >
+              <HealthApp />
+            </Route>
+            <Route exact path="/">
+              <Redirect to="/tab1" />
+            </Route>
+          </IonRouterOutlet>
+          <IonToolbar>
+            <img id="psu-logo" src="assets/logo.svg" slot="start" />
+            <div slot="end">
+              <NavLink id="home" text="Home" href="/tab1" />
+              <NavLink id="team" text="Team" href="/tab2" />
+              <NavLink id="profile" text="Profile" href="/profile">
+                <NavLink id="logs" text="Logs" href="/tab3" />
+              </NavLink>
+              <NavLink id="admin" text="Admin" href="/admin">
```

Comment:
Is there any logic implemented yet that will only show the admin link if the user is an admin?